preprocess.js

```diff
@@ -16,13 +16,24 @@ function getDirectories(srcpath) {
 
 const ROOT_DIR = `${__dirname}`;
 
+//alternate sort values for non-semver values
```

Comment:
this `versionMap` and `version` function look like some magic that could probably use a comment explaining WHY it's done.  For example:

```js
// This version map and version function allow versions that do not follow semver syntax to also
// be included in the version selection sorting for the site.  "local" is the developer version
// used when testing docs changes in a local development environment.  We set this "local"
// version as 7.7.7 (a high value) so that it will show up as the "latest" version in the site's
// version selection dropdown.
```

would you mind including something like that?

Reply:
done!